common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -19,9 +19,7 @@
         <span class="trailing_text">${trailing_text}</span>
 
         <span class="status" id="${id}_status" aria-describedby="label_${id}" data-tooltip="${status.display_tooltip}">
-            <span class="sr">
-                ${status.display_name}
-            </span>
+            <span class="sr">${status.display_tooltip}</span>
```

Comment:
The Lahore PR introduced these changes, and @alisan617 added the parent tag aria-describedby attributes in her PR. While resolving merge conflicts, the Lahore changes got lost (which thankfully had unit test coverage!).